packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -103,6 +103,8 @@ class _ModalBottomSheetLayout extends OneChildLayoutDelegate {
     childTop.end = childSize.height;
     return new Point(0.0, size.height - childTop.value);
   }
+
+  bool shouldRelayout() => true;
```

Comment:
I didn't pass the old delegate because I didn't understand why you'd want it. If the delegate changes, then layout happens anway.  And if oldDelegate == this then what's the point of the parameter?